File: fix_encoding.js
const fs = require('fs');

const files = [
    "c:\\Users\\diesy\\.gemini\\antigravity\\scratch\\diesyt_store\\index.html",
    "c:\\Users\\diesy\\.gemini\\antigravity\\scratch\\diesyt_store\\accounts.html"
];

const replacements = {
    '<span class="why-icon">???</span>': '<span class="why-icon">🛡️</span>',
    '<span class="why-icon">?</span>': '<span class="why-icon">⚡</span>',
    '<span class="why-icon">??</span>': '<span class="why-icon">💎</span>',
    '<div class="stat-num">? Instant</div>': '<div class="stat-num">⚡ Instant</div>',
    '&ccounts': 'Accounts',
    '&bout': 'About',
    '&LL': 'ALL',
    'V&LOR&NT': 'VALORANT',
    'INDI&\'S': "INDIA'S",
    'M&RKETPL&CE': 'MARKETPLACE',
    '&CCOUNTS': 'ACCOUNTS',
    'V&LOR&NT': 'VALORANT'
};

for (const file of files) {
    try {
        let content = fs.readFileSync(file, 'utf8');
        for (const [oldStr, newStr] of Object.entries(replacements)) {
            content = content.split(oldStr).join(newStr);
        }
        fs.writeFileSync(file, content, 'utf8');
    } catch (e) {
        console.error("Failed to read/write " + file + ": " + e.message);
    }
}
console.log("Replacement complete.");

File: fix_encoding.py
import os

files_to_fix = [
    r"c:\Users\diesy\.gemini\antigravity\scratch\diesyt_store\index.html",
    r"c:\Users\diesy\.gemini\antigravity\scratch\diesyt_store\accounts.html"
]

replacements = {
    '<span class="why-icon">???</span>': '<span class="why-icon">🛡️</span>',
    '<span class="why-icon">?</span>': '<span class="why-icon">⚡</span>',
    '<span class="why-icon">??</span>': '<span class="why-icon">💎</span>',
    '<div class="stat-num">? Instant</div>': '<div class="stat-num">⚡ Instant</div>',
    '&ccounts': 'Accounts',
    '&bout': 'About',
    '&LL': 'ALL',
    'V&LOR&NT': 'VALORANT',
    'INDI&\'S': "INDIA'S",
    'M&RKETPL&CE': 'MARKETPLACE',
    '&CCOUNTS': 'ACCOUNTS',
    'V&LOR&NT': 'VALORANT'
}

for file_path in files_to_fix:
    try:
        with open(file_path, 'r', encoding='utf-8', errors='ignore') as f:
            content = f.read()
    except Exception as e:
        print(f"Failed to read {file_path}: {e}")
        continue

    for old, new in replacements.items():
        content = content.replace(old, new)
        
    with open(file_path, 'w', encoding='utf-8') as f:
        f.write(content)

print("Replacement complete.")
